.azuredevops/modulePipelines/ms.operationsmanagement.solutions.yml

```diff
@@ -0,0 +1,54 @@
+name: 'OperationsManagement - Solutions'
+
+parameters:
+  - name: removeDeployment
+    displayName: Remove deployed module
+    type: boolean
+    default: true
+  - name: prerelease
+    displayName: Publish prerelease module
+    type: boolean
+    default: false
+
+pr: none
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '/.azuredevops/modulePipelines/ms.operationsmanagement.solutions.yml'
+      - '/.azuredevops/pipelineTemplates/module.*.yml'
```

Comment:
Depends on #1502

Reply:
True, changed to what proposed there, but I'm keeping the comment open to wait for confirmation from the other PR

Reply:
@eriqua, note the latest comment in #1502

Reply:
There is an issue with using `/*.yml` in case of GH